fixed a bug in CreateCertificate script (#11685)
demisto/content: Packs/EWS/ReleaseNotes/1_8_6.md

#### Scripts
##### CreateCertificate

> Fixed a bug that prevents the script from running with "The term 'New-SelfSignedCertificate' is not recognized as a name of a cmdlet" error message.

Reviewer: ```suggestion
Fixed an issue where the script was prevented from running by a "The term 'New-SelfSignedCertificate' is not recognized as a name of a cmdlet" error message.
```
